src/commands/index.ts

```diff
@@ -138,6 +138,7 @@ export function load(client: any): any {
   client.functions.secrets.prune = loadCommand("functions-secrets-prune");
   client.functions.secrets.set = loadCommand("functions-secrets-set");
   client.help = loadCommand("help");
+  client.help = loadCommand("gemini-help");
```

Comment:
Not needed for now since this is a draft/POC, but I definitely think we'd want to start this off behind an experiment